apps/modernization-ui/src/apps/search/investigation/transformer.ts

```diff
@@ -16,6 +16,10 @@ import { EventDate, Identification, InvestigationFilterEntry } from './Investiga
 import { Selectable, asValue, asValues } from 'options/selectable';
 
 export const transformObject = (data: InvestigationFilterEntry): InvestigationFilter => {
+    if (!data) {
+        return {};
```

Comment:
Actually we wont need this because the data is already mandatory so this if block is not needed